src/where.js

```diff
@@ -35,11 +36,27 @@ var _has = require('./internal/_has');
  *      pred({a: 'foo', b: 'bar', x: 11, y: 19}); //=> false
  *      pred({a: 'foo', b: 'xxx', x: 10, y: 19}); //=> false
  *      pred({a: 'foo', b: 'xxx', x: 11, y: 20}); //=> false
+ *
+ *      // Recursive behavior
+ *      var pred = R.where({
+ *        a: {
+ *          x: R.equals("foo")
+ *        },
+ *        b: R.complement(R.equals('bar'))
+ *      });
+ *
+ *      pred({a: { b: 'foo' }, b: 'xxx'}); //=> true
+ *      pred({a: { b: 'xxx' }, b: 'xxx'}); //=> false
+ *      pred({a: { b: 'foo' }, b: 'bar'}); //=> false
```

Comment:
Let's stick with `{k: v}` for the formatting of object literals (rather than `{ k: v }`).